front-end/src/App.css

```diff
@@ -0,0 +1,55 @@
+body{
+    background: rgb(239, 255, 217);
+}
+.post{
+    /* display: inline-grid;
+    grid-template-columns: auto auto auto ;
+    gap: 10px;
+    justify-content: center; */ 
+
+    /* border: 2px solid beige;
+    box-shadow: 2px 3px black;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap; */
+
+    /* justify-content: space-around; */
+
+    /* gap: 20px;
+    overflow-y: unset; */
+    
+}
```

Comment:
remove the `.post` css class since it is commented it out, also you need to have one empty line between each css block